docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -95,6 +102,52 @@ \section{Failure Mode and Effect Analysis}
   entered in the table, including strategies involving providing additional
 documentation, and/or test cases.}
 
+\begin{table}
```

Comment:
kinda weirdly centered, but maybe we fix it later because I couldn't find how to fix the centering. looks like this 
![Screenshot (834)](https://github.com/user-attachments/assets/a4cb6ad5-168c-497e-92c0-39e5a513736c)
too far justified to the left